### PR TITLE
Added logging to status

### PR DESCRIPTION
**Pull request by vastradparayya**
Signed-off-by: Parayya Vastrad <parayya.vastrad@seagate.com>

# Problem Statement
- Added Log.init to bundle_generate to log message and logging to status

# Design
-  NA

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]:  N
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/support_framework/bundle_generate.py [hunk 1]
```diff
@@ -112,6 +112,10 @@ async def init(command: List):
         command:        cli Command Object :type: command
         return:         None
         """
+        Conf.load('cortx_conf', 'json:///etc/cortx/cortx.conf')
+        log_level = Conf.get('cortx_conf', 'utils>log_level', 'INFO')
+        Log.init('support_bundle_node', '/var/log/cortx/utils/support/', \
+            level=log_level, backup_count=5, file_size_in_mb=5)
```
Comment (vastradparayya) [inline py-utils/src/utils/support_framework/bundle_generate.py:118]: Added Log.init to logging messages without syslogserver

File: py-utils/src/utils/support_framework/bundle_generate.py [hunk 2]
```diff
@@ -215,7 +219,10 @@ async def init(command: List):
 
         ComponentsBundle._create_summary_file(bundle_id, node_name, \
             comment, bundle_path)
-
+        # SB status logging
+        Log.init('support_bundle_status', '/var/log/cortx/utils/support/', \
+            level=log_level, backup_count=5, file_size_in_mb=5, \
+            syslog_server='localhost', syslog_port=514)
```
Comment (vastradparayya) [inline py-utils/src/utils/support_framework/bundle_generate.py:225]: Added Log.init to logging status messages with syslogserver

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**